source/jormungandr/jormungandr/street_network/geovelo.py

```diff
@@ -79,27 +79,35 @@ def _pt_object_summary_isochrone(cls, pt_object):
         return [coord.lat, coord.lon, getattr(pt_object, 'uri', None)]
 
     @classmethod
-    def _make_request_arguments_isochrone(cls, origins, destinations):
+    def _make_request_arguments_bike_details(cls, bike_speed_mps):
+        bike_speed = mps_to_kmph(bike_speed_mps)
+
+        return {
+            'profile': 'MEDIAN',  # can be BEGINNER, EXPERT
```

Comment:
Can't we use enum here to define the profile and bike type by any chance ?

Reply:
Not sure it's worth it as there is no way to manage it from Navitia's API.